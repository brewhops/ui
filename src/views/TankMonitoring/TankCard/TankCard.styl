.tank {
  background: Teal;
  width: 100%;
  height: 100%;
  min-height 128px;

  table {
    width: 100%;
    padding-left: 5px;
    padding-right: 7px;

    td:nth-child(2) {
      text-align: right;
    }

    td {
      padding: 3px;
    }
  }

  .tank-name {
    padding: 8px;
    font-size: 20px;
  }

  .tank-action {
    padding-left: 10px;
    padding-right: 10px;
    margin-top: 10px;
    margin-bottom: 8px;
    text-align: center;
  }
}