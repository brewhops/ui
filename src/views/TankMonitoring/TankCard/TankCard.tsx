import React from 'react';
import './TankCard.styl'

export interface ITankCardProps {
  tankName: string;
  beerName: string;
  pressure: number;
  batchName: string;
  actionName: string;
  status: string;
  temperature: number;
}

const TankCard: React.FC = () => {
  return (    
  <a>
    <div className="tank">
      <div className="tank-name">
        Tank
      </div>
      <table>
        <tr>
          <td>tank.beer_name</td>
          <td>tank.pressure psi</td>
          <td>tank.status</td>
        </tr>
        <tr>
          <td>tank.batch_name</td>
          <td>tank.temperature ÂºF</td>
        </tr>
      </table>

      <div className="tank-action">
        tank.action_name
      </div>
    </div>
  </a>
  )
}

export default TankCard;