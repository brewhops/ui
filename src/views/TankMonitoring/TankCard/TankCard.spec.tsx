import React from 'react';
import TankCard from './TankCard';
import { shallow } from 'enzyme';

describe('src/pages/TankMonitoring/TankCard/TankCard.tsx', () => {
  it('renders a tank card', () => {
    expect(shallow(<TankCard/>)).toMatchSnapshot();
  })
});