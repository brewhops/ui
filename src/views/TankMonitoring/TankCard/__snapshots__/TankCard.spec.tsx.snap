// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/pages/TankMonitoring/TankCard/TankCard.tsx renders a tank card 1`] = `ShallowWrapper {}`;
